src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -285,9 +285,11 @@ public void periodic() {
 
                 if(balanceTarget != null ) {
                         if (Math.abs(getGyroscopePitch().getDegrees()) >= AUTO_BALANCE_TOLERANCE_DEGREES) {
-                                chassisSpeeds.vyMetersPerSecond = yPID.calculate(getGyroscopePitch().minus(balanceTarget).getRadians());
+                                chassisSpeeds.vyMetersPerSecond = yPID.calculate(-getGyroscopePitch().minus(balanceTarget).getDegrees());
+                                chassisSpeeds.vxMetersPerSecond = xPID.calculate(-getGyroscopePitch().minus(balanceTarget).getDegrees());
                         } else if (Math.abs(getGyroscopeRoll().getDegrees()) >= AUTO_BALANCE_TOLERANCE_DEGREES) {
-                                chassisSpeeds.vxMetersPerSecond = xPID.calculate(getGyroscopeRoll().minus(balanceTarget).getRadians());
+                                chassisSpeeds.vyMetersPerSecond = yPID.calculate(-getGyroscopeRoll().minus(balanceTarget).getDegrees());
+                                chassisSpeeds.vxMetersPerSecond = xPID.calculate(-getGyroscopeRoll().minus(balanceTarget).getDegrees());
                         }
 
```

Comment:
This is the same as my comment on the closed PR. Was thinking we need an else that sets the speeds to 0 if within both tolerances